docs/release-notes/8.0/8.0.35-30.0.md

```diff
@@ -0,0 +1,31 @@
+# Percona XtraBackup 8.0.35-30 (2023-12-04)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release merges the MySQL 8.0.35 code base.
+
+## Bug fix
+
+* [PXB-3003](https://jira.percona.com/browse/PXB-3003): Percona XtraBackup discovering redo logs to parse and the server purging redo logs simultaneously could cause a race condition.
+
+* [PXB-3139](https://jira.percona.com/browse/PXB-3139): Fix for gcc-13 and clang-16 compilation issues.
+
+* [PXB-3147](https://jira.percona.com/browse/PXB-3147): Percona XtraBackup failed to execute the `DO innodb_redo_log_consumer_register("PXB");` query.
+
+* [PXB-3181](https://jira.percona.com/browse/PXB-3181): The use of the `innodb-use-native-aio=true` option resulted in printing duplicate timestamps.
+
+* [PXB-3173](https://jira.percona.com/browse/PXB-3173): The xbcloud was refactored to use multiple threads with a global list of files. This list did not allow users to download a subset of files.
+
+* [PXB-3141](https://jira.percona.com/browse/PXB-3141): Adjusted folder creation on `ds_fifo.cc` and xbcloud to use 777 as permission.
```

Comment:
777 here can be misinterpreted as too open, which would be true for a file, but not for a folder. I would adjust this to:

Adjusted folder creation on `ds_fifo.cc` and xbcloud to use proper permission.